### PR TITLE
SUP-437 Commenting out the the `-Xcheckinit` compiler flag

### PR DESCRIPTION
**Pull request by gpolumbo-broad**
**PR checklist**
- [ ] For each library you've modified here, decide whether it requires a major, minor, or no version bump. (Click [here](/broadinstitute/workbench-libs/blob/develop/CONTRIBUTING.md) for guidance)

If you're doing a **major** or **minor** version bump to any libraries:
- [ ] Bump the version in `project/Settings.scala` `createVersion()`
- [ ] Update `CHANGELOG.md` for those libraries
- [ ] I promise I used `@deprecated` instead of deleting code where possible

In all cases:
- [ ] Replace the appropriate version hashes in `README.md` and the `CHANGELOG.md` for any libs you changed with `TRAVIS-REPLACE-ME` to auto-update the version string
- [ ] Get two thumbsworth of PR review
- [ ] Verify all tests go green (CI _and_ coverage tests)
- [ ] Squash commits and **merge to develop**
- [ ] Delete branch after merge

### CODE REVIEW

**Review by s-rubenstein — APPROVED** [review]
LGTM

**Review by dvoet — APPROVED** [review]

**Comment by Qi77Qi** [comment]
can you update https://github.com/broadinstitute/workbench-libs/blob/develop/notifications/CHANGELOG.md as well

File: notifications/CHANGELOG.md [hunk 1]
```diff
@@ -2,6 +2,14 @@
 
 This file documents changes to the `workbench-notifications` library, including notes on how to upgrade to new versions.
 
+## 0.4
+
+### Changed
```
Comment (rtitle) [inline notifications/CHANGELOG.md]: Text looks good but think this should go in the `0.3` section if we're not bumping versions

File: notifications/CHANGELOG.md [hunk 2]
```diff
@@ -2,6 +2,14 @@
...
+- Modified compiler scala 2.12 and 2.13 compiler settings and commented out: 
+  `"-Xcheckinit", // Wrap field accessors to throw an exception on uninitialized access.`
+
+SBT dependency: `"org.broadinstitute.dsde.workbench" %% "workbench-util" % "0.4-TRAVIS-REPLACE-ME"`
+
 ## 0.3
 
 SBT dependency: `"org.broadinstitute.dsde.workbench" %% "workbench-util" % "0.3-426a0c2"`
```
Comment (rtitle) [inline notifications/CHANGELOG.md]: Also change this hash to `TRAVIS-REPLACE-ME`

**Review by rtitle — APPROVED** [review]
:shipit:

**Pull request merged**